tests/koji/test_koji_vmi.py

```diff
@@ -1,6 +1,7 @@
 import os
 
 from pytest import raises, mark
+from unittest.mock import patch, mock_open
```

Comment:
These imports aren't used, can you please clean them up?

Reply:
removed